app/models/chargeback_container_project.rb

```diff
@@ -62,19 +64,38 @@ def self.build_results_for_report_ChargebackContainerProject(options)
       @data_index.store_path(:container_project, :by_group_id, g.id, g.container_project || g.old_container_project)
     end
 
+    if @options[:groupby_tag]
+      @tags_for_category = Classification.find_by_name(@options[:groupby_tag]).entries
+      @tags_for_category.each do |tag|
+        ContainerProject.with_any_tags([tag.tag_id]).each do |project|
+          @data_index.store_path(:tag, :by_project_id, project.id, tag)
+        end
+      end
+    end
```

Comment:
@kbrock Some help here? 
The hash im trying to build is `project_id => tag` so if all projects have a `Department=x` tag ill know the x for each project.